server/controllers/finance/reports/reportAccounts/report.handlebars

```diff
@@ -0,0 +1,54 @@
+{{> head title="PATIENT_RECORDS.FINANCIAL_ACTIVITY.REPORT_TITLE" }}
+
+<body>
+
+  {{> header }}
+
+  <h3 class="text-center text-uppercase"><strong>{{translate "REPORT.ACCOUNT"}}</strong></h3>
+  <h4 class="text-center"><strong>{{ title.accountNumber }}</strong></h4>
+  <h4 class="text-center text-uppercase"><strong>{{ title.accountLabel}}</strong></h4>
+  
+  <section>
+    <table class="table table-condensed table-striped table-report table-bordered">
+      <thead>
+        <tr class="text-capitalize text-center" style="background-color: #ddd;">
+          <th style="width: 10%;">{{translate "TABLE.COLUMNS.INVOICE_ID" }}</th>
```

Comment:
Only class 4 accounts will have invoice IDs.  I propose you remove this column for a generic account report.